### ISSUE TITLE
Other players can't see you dismiss your nano

### ISSUE THREAD

**Issue by gsemaj** [OpenFusionProject/OpenFusion#83]
Likely due to this sanity check in `NanoManager::summonNano`:
```
if (slot > 2 || slot < 0) // TODO: implement proper way to dismiss nano if slot = -1
        return; // sanity check
```

**Comment by dongresource** [comment]
I just now encountered this bug while testing something else and thought it was my changes. :P
I'll fix it.

**Comment by dongresource** [comment]
Forgot to close this yesterday. Fixed by c1ac2250a068a31306bbe8caefb79ef0dd21806d.
